resources/views/account/edit.blade.php

```diff
@@ -96,3 +110,53 @@
         {{ Form::close() }}
     </div>
 @stop
+
+@push('scripts')
+<script type="text/javascript">
+var autocomplete;
+var componentForm = {
+    neighborhood: 'long_name', // city within borough (Williamsburg)
```

Comment:
Is this "neighborhood within borough" not "city within borough"?  I don't know if this is a typo in your comment or if it's just confusing :)